tests/ports/qemu/mpy_arch_flags_test.py

```diff
@@ -0,0 +1,128 @@
+try:
+    import sys, io, vfs
+
+    sys.implementation._mpy
+    io.IOBase
+except ImportError:
```

Comment:
This will need to be `(ImportError, AttributeError)` to catch the case of `._mpy` or `io.IOBase` not existing.